dist/save/index.js

```diff
@@ -5350,6 +5350,14 @@ function isValidEvent() {
     return constants_1.RefKey in process.env && Boolean(process.env[constants_1.RefKey]);
 }
 exports.isValidEvent = isValidEvent;
+function getInputAsArray(name, options) {
+    return core
+        .getInput(name, options)
+        .split("\n")
+        .map(s => s.trim())
```

Comment:
I suppose this gets rid of the "\r" on Windows as well, which I believe was the issue observed in https://github.com/actions/cache/issues/330#issuecomment-634505960

Reply:
Yes, I also added a test for this case.